config/default.json

```diff
@@ -101,5 +101,12 @@
     "birminghamEmail": "IA_HEARING_CENTRE_BIRMINGHAM_EMAIL",
     "hattonCrossEmail": "IA_HEARING_CENTRE_HATTON_CROSS_EMAIL",
     "glasgowEmail": "IA_HEARING_CENTRE_GLASGOW_EMAIL"
+  },
+  "pcq": {
+    "url": "https://pcq.aat.platform.hmcts.net",
+    "health": "/health",
+    "path": "/service-endpoint",
+    "returnUrl": "localhost:3000/about-appeal",
+    "tokenKey": "bnrfX2S5iS6VSC9KY1lRmkA5ixSQFMqHFumNO4JDUkmns4PZyiBkJIMcpT8wWfC"
```

Comment:
let's not hardcode the key here. 
```
pcq": {
    "url": "PCQ_URL",
    "health": "/health",
    "path": "/service-endpoint",
    "returnUrl": "PCQ_RETURN_URL",
    "tokenKey": "PCQ_TOKEN_KEY"
```

and for running locally, set a local environment variable to set the right value locally.
i.e.
```
export PCQ_URL= ....
export PCQ_RETURN_URL= ...
export PCQ_TOKEN_KEY = ....

```